src/app.ts

```diff
@@ -577,6 +587,7 @@ export class App {
 
     const rawMethods = processResource(api);
 
+    // tslint:disable-next-line:no-any
     const methods = rawMethods.map((x: any) => ({
```

Comment:
After `processResource()` fix, we can use `RestMethod` here as well:
```js
const methods = rawMethods.map((x: RestMethod) => ({
```
note, that we may want to do this also:
```js
namespace: getNamespace(x.id!),
```
(add `!` to say that we're sure that `id` is not undefined)

Reply:
Per the comment above, `methods` is not actually used, so I'm going to remove this.